docs/design/job-resource-reservation-design.md

```diff
@@ -5,6 +5,7 @@
 ## Deprecated
 
 The `Reservation` plugin has been deleted. This doc is deprecated. It will be deleted in the future.
+`Reservation` is replaced by [SLA](sla-plugin.md).
```

Comment:
I'd like to know what was the original reservation feature like? SLA plugin is related with reservation?

Reply:
They all try to solve the problem of hunger in large vcjob.
BTW, I don't know why the reservation plugin is removed